src/docker/configs.ts

```diff
@@ -32,7 +32,7 @@ export function getSocketPath(): string {
   } else {
     path = path ? path : DEFAULT_UNIX_SOCKET_PATH;
   }
-  logger.debug("using docker socket path:", { socketPath: path });
+  logger.trace("using docker socket path:", { socketPath: path });
```

Comment:
These are annoying but helpful if we need to debug local Docker requests. May remove entirely in a future PR.